From 72566eda43b97129b30916fe20750df50f7b6fb0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Fri, 12 Jul 2019 14:58:42 +0200
Subject: [PATCH 006/152] messageTray: Remove unused source object parameter

This is has been added in commit b150869b5 but is never used

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/700
---
 js/ui/messageTray.js | 1 -
 1 file changed, 1 deletion(-)

diff --git a/js/ui/messageTray.js b/js/ui/messageTray.js
index 4e1841fc0..6a65d5feb 100644
--- a/js/ui/messageTray.js
+++ b/js/ui/messageTray.js
@@ -1004,7 +1004,6 @@ var MessageTray = class MessageTray {
 
     _addSource(source) {
         let obj = {
-            source: source,
             notifyId: 0,
             destroyId: 0,
         };
-- 
2.23.0

